test/functional/wallet_multiwallet.py

```diff
@@ -212,6 +230,17 @@ def wallet_file(name):
         w2 = node.get_wallet_rpc(wallet_names[1])
         w2.getwalletinfo()
 
+        self.log.info("Concurrent wallet loading")
+        threads = []
+        for x in range(3):
+            t = Thread(target=test_load_unload, args=(node, wallet_names[2], x, ))
+            t.start()
+            threads.append(t)
+        for t in threads:
+            t.join()
+        global got_loading_error
+        assert_equal(got_loading_error, True)
```

Comment:
I don't think we can assert that a race happened. Races are only intermittent.

Reply:
I think this one is pretty much guaranteed, loading takes a bit.